src/dmd/expression.d

```diff
@@ -1834,9 +1834,9 @@ extern (C++) abstract class Expression : RootObject
             loc = e.loc;
 
         if (e.op == TOK.type)
-            error("%s `%s` is a type, not an lvalue", e.type.kind(), e.type.toChars());
+            error("cannot modify `%s`. It is a %s", e.type.toChars(), e.type.kind());
```

Comment:
Not sure about this, maybe "cannot modify `%s`, it is a %s`" is better?

Reply:
Ah, see my comment https://github.com/dlang/dmd/pull/8009#discussion_r173763618

Reply:
Good idea. Thanks!